### PR TITLE
Migrate package_controller_test.rb (diff, revision, derived packages)

### PR DESCRIPTION
**Pull request by mdeniz**
Migrate functional tests to RSpec:

- test_diff_is_empty
- test_revision_is_empty
- test_derived_packages

### CODE REVIEW

**Comment by mdeniz** [comment]
[(2) Migrate test/functional/webui/package_controller_test.rb (tests for the plain view, diff, revision, derived packages) to the new test suite](https://trello.com/c/IPnlPGRf/27-2-migrate-test-functional-webui-package-controller-test-rb-tests-for-the-plain-view-diff-revision-derived-packages-to-the-new-te)

**Comment by hennevogel** [comment]
Please rebase

**Comment by hennevogel** [comment]
Otherwise it looks good to me :-)

**Comment by mdeniz** [comment]
Done

File: src/api/spec/controllers/webui/package_controller_spec.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
-require 'rails_helper'
 require "webmock/rspec"
+require 'rails_helper'
```
Comment (mdeniz) [inline src/api/spec/controllers/webui/package_controller_spec.rb]: I didn't... was as a result of a rebase... but I doesn't change anything at all.. right?

**Pull request merged**